src/test/java/uk/gov/hmcts/reform/unspec/utils/ElementUtilsTest.java

```diff
@@ -61,4 +62,18 @@ void shouldReturnEmptyListIfListOfElementIsNull() {
             assertThat(unwrapElements(null)).isEmpty();
         }
     }
+
+    @Nested
+    class BuildElement {
+
+        @Test
+        void shouldBuildElement_whenObjectIsProvided() {
+            CaseDocument document = CaseDocument.builder().build();
+
+            Element<CaseDocument> element = ElementUtils.element(document);
+
+            assertThat(element.getId()).isNotNull();
+            assertThat(element.getValue()).isEqualTo(document);
+        }
+    }
```

Comment:
could write this test with a String instead of CaseDocument but not super important